src/coreclr/jit/llvmcodegen.cpp

```diff
@@ -1303,6 +1307,13 @@ void Llvm::buildReturn(GenTree* node)
     }
     else
     {
+        retValValue = getGenTreeValue(retValNode);
```

Comment:
we are reusing `retValValue` below, nothing incorrect about that, just wanted to ask if it was deliberate and just saves a variable?

Reply:
Yes, it is; the code is temporary and will go away in #2098 so I didn't feel it needed more than this.